README.md

```diff
@@ -1,24 +1,199 @@
+- [📗 Table of Contents](#-table-of-contents)
+- [Skill Habour ](#skill-habour-)
+- [🛠 Built With ](#-built-with-)
+  - [Tech Stack ](#tech-stack-)
+- [💮 Key Features ](#-key-features-)
+- [🚀 Important Links ](#-important-links-)
+    - [FRONTEND REPOSITORY](#frontend-repository)
+    - [KANBAN BOARD](#kanban-board)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites ](#prerequisites-)
+  - [🧰 Setup ](#-setup-)
+  - [🔧 Install ](#-install-)
+  - [🪜 Usage ](#-usage-)
+- [🧑🏽‍💻 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+      - [⭐️⭐️⭐️⭐️ Star the repository 💸](#️️️️-star-the-repository-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# Skill Habour <a name="about-project"></a>
 
-* Database initialization
+**🎓 Skill Habour** is a cheap platform that helps you  dive into a sea of knowledge without breaking the bank! 🌊 Skill Harbour is your secret treasure trove for affordable courses that won't leave your wallet high and dry.
 
-* How to run the test suite
+# 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+## Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">ReactJS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+# 💮 Key Features <a name="key-features"></a>
+
+- **User can authenticate**
+- **User can browse a list of courses**
+- **User can reserve or enroll in a course**
+- **User can check their reservations**
+- **The application is responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+# 🚀 Important Links <a name="important-links"></a>
+
+<!-- ### [LIVE DEMO]() -->
+### [FRONTEND REPOSITORY](https://github.com/coderacheal/skill-habour-fe)
+### [KANBAN BOARD](https://github.com/users/coderacheal/projects/5/views/1)
+<!-- ### [API DOCUMENTATION]() -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+# 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+## Prerequisites <a name="prerequisites"></a>
+
+  
+## 🧰 Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/coderacheal/skill-habour-be.git
+```
+
+## 🔧 Install <a name="install"></a>
+
+Install this project with:
+
+```sh
+  cd skill-habour-be
+  
+  bundle install
+
+  rails db:create
```

Comment:
- [ ] When I try to create the database, this happens:

![image](https://github.com/coderacheal/skill-habour-be/assets/57421823/69ba37b8-78a5-47e5-8b4e-11b86bc2e35e)

This is because the master key is missing, and your users need to create it locally.

Kindly, add these instructions to your readme:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://github.com/coderacheal/skill-habour-be/assets/57421823/3c817daf-f611-439d-8330-c1580d698625)


That way, your users will be able to create the database. 💪